doc/cephadm/osd.rst

```diff
@@ -363,6 +363,52 @@ Example command:
     device.  To disable this behavior, see :ref:`cephadm-osd-declarative`.
 
 
+.. _osd_autotune:
+
+Automatically tuning OSD memory
+===============================
+
+OSD daemons will adjust their memory consumption based on the
+:confval:`osd_memory_target` config option (several gigabytes, by
+default).  If Ceph is deployed on dedicated nodes that are not sharing
+memory with other services, cephadm can automatically adjust the per-OSD
+memory consumption based on the total amount of RAM and the number of deployed
+OSDs.
+
+This option is enabled globally with::
+
+  ceph config set osd osd_memory_target_autotune true
+
+Cephadm will start with a fraction
+(:confval:`mgr/cephadm/autotune_memory_target_ratio`, which defaults to
```

Comment:
i am afraid that we don't yet have module options extracted automatically at this moment. i can implement it.

Reply:
before that we have to define an option like https://github.com/ceph/ceph/blob/master/doc/rados/configuration/general-config-ref.rst . see `max_open_files`.

Reply:
ok, leaving it off for now